backend/api/src/handlers/types/client-registry-response.ts

```diff
@@ -0,0 +1,23 @@
+export type clientRegistryResponse = {
+  client_name: string;
+  public_key: string;
+  public_key_source: string;
+  redirect_uris: string[];
+  contacts: string[];
+  scopes: string[];
+  subject_type: string;
+  service_type: string;
+  sector_identifier_uri: string;
+  client_id: string;
+  post_logout_redirect_uris: string[];
+  back_channel_logout_uri: null | string;
+  token_endpoint_auth_method: string;
+  response_type: string;
+  jar_validation_required: boolean;
+  claims: string[];
+  client_type: string;
+  id_token_signing_algorithm: "ES256" | "RS256";
+  jwks_uri: null | string;
+  channel: string;
+  max_age_enabled: boolean;
```

Comment:
These two are new - dont know if this expected elsewhere in the code

Reply:
The current code is quite permissive - these will have been stored if they are included in the response. But good we have this type, we should try and use it in most places in the codebase!